guides/app-guides/univ3-lp.md

```diff
@@ -0,0 +1,77 @@
+---
+description: Details on Angle UniV3 incentive mechanism for LPs
+---
+
+# UniV3 LP incentives in Angle
+
+Angle implemented a specific mechanism to reward UniV3 liquidity providers according to their added value to the protocol. Basically, it allows to reward LPs more granularily according to the efficiency of the liquidity they provide. In turn, they have the opportunity to provide liquidity in more complex ways that benefit from all Uniswap V3 possibilities.
```

Comment:
Angle implements a specific mechanism to reward Uniswap V3 liquidity

Let's make sure to explain all abbreviations used in the guide (so that a newbie can understand to start with)

Reply:
Implement means "mettre en place", so it is to be used at the past tense here.